### ISSUE TITLE
[1.16.4] Crash when trying to attune to constellation

### ISSUE THREAD

**Issue by SuluCandles** [HellFirePvP/AstralSorcery#1660]
When walking into an attunement altar, the game freezes and crashes. Snippet from the crash log:

`Time: 1/18/21 5:22 PM
Description: Saving entity NBT

java.lang.NullPointerException: Saving entity NBT
	at com._idrae.travelers_index.data.ItemChecklistStorage.writeNBT(ItemChecklistStorage.java:32) ~[?:1.16.4-1.0.2] {re:classloading}
	at com._idrae.travelers_index.data.ItemChecklistStorage.writeNBT(ItemChecklistStorage.java:16) ~[?:1.16.4-1.0.2] {re:classloading}
	at com._idrae.travelers_index.data.ItemChecklistProvider.serializeNBT(ItemChecklistProvider.java:29) ~[?:1.16.4-1.0.2] {re:classloading}
	at com._idrae.travelers_index.data.ItemChecklistProvider.serializeNBT(ItemChecklistProvider.java:14) ~[?:1.16.4-1.0.2] {re:classloading}
	at net.minecraftforge.common.capabilities.CapabilityDispatcher.serializeNBT(CapabilityDispatcher.java:127) ~[?:?] {re:classloading}
	at net.minecraftforge.common.capabilities.CapabilityProvider.serializeCaps(CapabilityProvider.java:86) ~[?:?] {re:computing_frames,re:mixin,re:classloading}
	at net.minecraft.entity.Entity.func_189511_e(Entity.java:1408) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,xf:fml:astralsorcery:add_custom_entity_collision,pl:mixin:APP:spiderstpo.mixins.json:EntityMixin,pl:mixin:APP:abnormals_core.mixins.json:EntityMixin,pl:mixin:APP:appliedenergistics2.mixins.json:spatial.EntityMixin,pl:mixin:APP:structure_gel.mixins.json:EntityMixin,pl:mixin:APP:immersiveengineering.mixins.json:accessors.EntityAccess,pl:mixin:APP:quark.mixins.json:EntityMixin,pl:mixin:A}
	at hellfirepvp.astralsorcery.client.util.camera.CameraTransformerPlayerFocus.onStartTransforming(CameraTransformerPlayerFocus.java:40) ~[?:1.16.4-1.13.9] {re:classloading}
	at hellfirepvp.astralsorcery.client.util.camera.ClientCameraManager.tick(ClientCameraManager.java:45) ~[?:1.16.4-1.13.9] {re:classloading}
	at hellfirepvp.observerlib.common.util.tick.TickManager.renderTick(TickManager.java:84) ~[?:1.16.4-1.5.1] {re:classloading}
	at hellfirepvp.observerlib.common.util.tick.TickManager$$Lambda$4628/1495630836.accept(Unknown Source) ~[?:?] {}
	at net.minecraftforge.eventbus.EventBus.doCastFilter(EventBus.java:247) ~[eventbus-3.0.5-service.jar:?] {}
	at net.minecraftforge.eventbus.EventBus.lambda$addListener$11(EventBus.java:239) ~[eventbus-3.0.5-service.jar:?] {}
	at net.minecraftforge.eventbus.EventBus$$Lambda$2891/581887257.invoke(Unknown Source) ~[?:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:297) ~[eventbus-3.0.5-service.jar:?] {}
	at net.minecraftforge.fml.hooks.BasicEventHooks.onRenderTickStart(BasicEventHooks.java:75) ~[?:?] {re:classloading}
	at net.minecraft.client.Minecraft.func_195542_b(Minecraft.java:974) [?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mining_helmet.mixins.json:MinecraftMixin,pl:mixin:APP:immersiveengineering.mixins.json:accessors.client.MinecraftAccess,pl:mixin:APP:charm.mixins.json:accessor.MinecraftAccessor,pl:mixin:APP:assets/botania/botania.mixins.json:AccessorMinecraft,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:607) [?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mining_helmet.mixins.json:MinecraftMixin,pl:mixin:APP:immersiveengineering.mixins.json:accessors.client.MinecraftAccess,pl:mixin:APP:charm.mixins.json:accessor.MinecraftAccessor,pl:mixin:APP:assets/botania/botania.mixins.json:AccessorMinecraft,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:184) [?:?] {re:classloading,re:mixin,pl:runtimedistcleaner:A,pl:mixin:A,pl:runtimedistcleaner:A}
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_51] {}
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_51] {}
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_51] {}
	at java.lang.reflect.Method.invoke(Method.java:497) ~[?:1.8.0_51] {}
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(FMLClientLaunchProvider.java:51) [forge-1.16.4-35.1.13.jar:35.1] {}
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider$$Lambda$479/2067586671.call(Unknown Source) [forge-1.16.4-35.1.13.jar:35.1] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-8.0.6.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54) [modlauncher-8.0.6.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72) [modlauncher-8.0.6.jar:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:82) [modlauncher-8.0.6.jar:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:66) [modlauncher-8.0.6.jar:?] {}
`

**Comment by Jeuvke** [comment]
Duplicate of #1594.
TL;DR: it's Traveller's Index's fault
